dashboard/app/views/layouts/_small_footer.html.haml

```diff
@@ -0,0 +1,39 @@
+.small-footer
+  = form_tag(locale_url, method: :post, id: 'localeForm', style: 'margin-bottom: 0px;') do
+    = hidden_field_tag :return_to, request.url
+    = select_tag :locale, options_for_select(options_for_locale_select, locale), onchange: 'this.form.submit();'
+  %small
+    -# TODO: Localize
+    = link_to 'Copyright', '#', class: 'copyright-link'
+    &nbsp;&nbsp;|&nbsp;&nbsp;
+    -# TODO: Localize
+    <a href="#" class="more-link">More <i class="fa fa-caret-up"></i></a>
+
+#copyright-flyout
+  -# TODO: Localize all of this?
+  %p
+    We thank our <a href="https://code.org/about/donors">donors</a>,
+    <a href="https://code.org/about/partners">partners</a>,
+    our <a href="https://code.org/about/team">extended team</a>,
+    our video cast, and
+    our <a href="https://code.org/about/advisors">education advisors</a>
+    for their support in creating Code Studio.
+  %p
+    We especially want to recognize the engineers from Google,
+    Microsoft, Facebook, Twitter, and many others who helped create
+    these materials.
+  %p
+    Ice Age&#8482; &amp; &copy; #{Time.now.year} 20th Century Fox. All Rights Reserved.
+    <br />AngryBirds&#8482; &amp; &copy; 2009-#{Time.now.year} Rovio Entertainment Ltd. All Rights Reserved.
+    <br />Plants vs. Zombies&#8482; &amp; &copy; #{Time.now.year} Electronic Arts Inc. All Rights Reserved
+  %p
+    Powered by Amazon Web Services
+  %p
+    &copy; Code.org, #{Time.now.year}. Code.org&reg;, the CODE logo and Hour of Code&reg;
+    are trademarks of Code.org.
+
+%ul#more-menu
+  %li= link_to 'Support', 'https://support.code.org'
+  %li= link_to 'Help us translate to your language', 'http://eepurl.com/lm_ln'
```

Comment:
http://eepurl.com/Im_In

I think you used the letter L but they're both i like "I'm in!"